src/main/java/com/uber/cadence/internal/worker/WorkflowWorker.java

```diff
@@ -243,7 +263,7 @@ private void sendReply(
     private final Duration retryServiceOperationInitialInterval = Duration.ofMillis(200);
     private final Duration retryServiceOperationMaxInterval = Duration.ofSeconds(4);
     private final Duration paginationStart = Duration.ofMillis(System.currentTimeMillis());
-    private Duration decisionTaskStartToCloseTimeout;
+    private Duration decisionTaskStartToCloseTimeout = Duration.ofSeconds(60);
```

Comment:
Removed the initialization.
Currently DecisionTaskWithHistoryIteratorImpl does not have access to WorkflowExecutionStarted event at construction time. Replay decider will have to pass it in.
Options:
1. Refactor the interface DecisionTaskHandler to not take a dependency on HistoryIterator and just consume DecisionTask instead. Because of how queries work and ReplayDecisionTaskWithHistoryIterator work this is a bit more involved refactor.

2. Pass WorkflowExecutionStarted at getHistory(). However, the requirement to pass this in is implementation specific and it doesn't seem appropriate to enforce it at the interface.

3. provide a separate method to pass it. This feels like a bad pattern as now it requires the user of this class to know that they need to call this method prior to calling getHistory();

4. Current solution. Abstracted it out in a method and only call that method in get history if we need to paginate. Best option IMO is 1. but I would rather do that in a separate CR